cli/internal/cmd/apply.go

```diff
@@ -40,6 +40,32 @@ import (
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 )
 
+// NewApplyCmd creates the apply command.
+func NewApplyCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "apply",
+		Short: "Apply a configuration to a Constellation cluster",
+		Long:  "Apply an upgrade to a Constellation cluster by applying the chosen configuration.",
+		Args:  cobra.NoArgs,
+		RunE:  runApply,
+	}
+
+	cmd.Flags().Bool("conformance", false, "enable conformance mode")
+	cmd.Flags().Bool("skip-helm-wait", false, "install helm charts without waiting for deployments to be ready")
+	cmd.Flags().Bool("merge-kubeconfig", false, "merge Constellation kubeconfig file with default kubeconfig file in $HOME/.kube/config")
```

Comment:
should we mark flags that are only relevant for init?

Reply:
I'm not sure how I feel about this, because the goal is to remove the distinction between `upgrade apply` and `init`.
Ideally, we would instead update the `apply` command to offer the functionality of the `init` specific flags also when running it to just upgrade the cluster.

Reply:
I also think that eventually all flags that were originally init-only should become modifiable through upgrade. Unless there is a specific reason why this is dangerous. But in the meantime it seems like a nice idea to me to have a note like `applies to phase X`.

Reply:
i went over the flags again and don't see any critical problem by not having the hint. is fine by me.